### ISSUE TITLE
Instance of 'UUID' has no 'int' member (no-member)

### ISSUE THREAD

**Issue by crbunney** [PyCQA/pylint#961]
The UUID.int attribute exists as per the [docs](https://docs.python.org/2/library/uuid.html#uuid.UUID.int), but appears to be set dynamically in the UUID class' `__init__` (if I've been looking at the correct [code](https://github.com/python/cpython/blob/d034cd6b2fdb79f7fe3a8fbfe23b4ce8c7b44130/Lib/uuid.py#L184)):

``` python
self.__dict__['int'] = int
```

This seems to cause pylint to generate an Error message when it shouldn't
### Steps to reproduce

I created this test file:

``` python
import uuid

u = uuid.UUID('{12345678-1234-5678-1234-567812345678}')
print u.int
```

I ran this command on it: `pylint -r n --unsafe-load-any-extension y --errors-only test.py`
This output was generated:

```
No config file found, using default configuration
************* Module test
E:  4, 6: Instance of 'UUID' has no 'int' member (no-member)
```
### Current behavior

The call to `u.int` is flagged as a potential missing member
### Expected behavior

No error raised
### pylint --version output

```
No config file found, using default configuration
pylint 1.5.6,
astroid 1.4.6
Python 2.7.11 (default, Jan 22 2016, 08:29:18)
[GCC 4.2.1 Compatible Apple LLVM 7.0.2 (clang-700.1.81)]
```
### pip freeze

```
astroid==1.4.6
colorama==0.3.7
lazy-object-proxy==1.2.2
pylint==1.5.6
six==1.10.0
wrapt==1.10.8
```

**Comment by PCManticore** [comment]
Thank you for reporting this issue. We do not currently understand modifications of the instance's dictionary, as it is the case for what UUID is doing, but this can be solved in the mean time through an astroid brain tip, through which we can inject the attribute in the AST.
